src/Server/TCPHandler.cpp

```diff
@@ -57,13 +57,31 @@
 #include <Common/config_version.h>
 
 using namespace std::literals;
+using namespace DB;
```

Comment:
Why not to place `convertToQueryParameters` to anonymous namespace (or just static) inside `namespace DB`?

Reply:
it is in anonymous ns. this `using` allows to not write `DB::` specifier for types used in this function.
e.g. `NameToNameMap convertToQueryParameters(const Settings & passed_params)`
instead of `DB::NameToNameMap convertToQueryParameters(const DB::Settings & passed_params)`